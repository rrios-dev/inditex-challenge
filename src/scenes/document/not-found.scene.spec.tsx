import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

import FavsProvider from "@/pods/favs/favs-provider";

import HeroFavsScene from "../hero-favs/hero-favs.scene";

describe("Not found scene", () => {
  it("Mount scene", () => {
    const { asFragment } = render(
      <FavsProvider>
        <HeroFavsScene />
      </FavsProvider>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
