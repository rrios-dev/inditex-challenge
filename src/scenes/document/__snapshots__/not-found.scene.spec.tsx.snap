// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Not found scene > Mount scene 1`] = `
<DocumentFragment>
  <main
    class="_hero-favs_06173e _flex_6eb20e _dir-column_6eb20e _gap-6_6eb20e"
  >
    <h1
      class="sr-only"
    >
      Hero Favs
    </h1>
    <div
      class="_flex_6eb20e _dir-column_6eb20e _gap-4_6eb20e"
    >
      <div
        class="_search-input-container_2e7b85"
      >
        <img
          alt="search item"
          data-nimg="1"
          decoding="async"
          height="12"
          loading="lazy"
          src="/icons/search.svg"
          style="color: transparent;"
          width="12"
        />
        <label
          class="sr-only"
          for=":r0:"
        >
          Search
        </label>
        <input
          class="_search-input_2e7b85"
          id=":r0:"
          placeholder="Search"
          type="text"
        />
      </div>
      <p
        class="_typography_142002 _variant-caption_142002 color-fg"
      >
        0 RESULT
      </p>
    </div>
    <ul
      class="_hero-selector_1d6cb9"
    >
      <li
        data-block="true"
      />
      <li
        data-block="true"
      />
      <li
        data-block="true"
      />
      <li
        data-block="true"
      />
      <li
        data-block="true"
      />
      <li
        data-block="true"
      />
      <li
        data-block="true"
      />
    </ul>
  </main>
</DocumentFragment>
`;
